app/src/main/res/layout/fragment_subscribers_detail.xml

```diff
@@ -144,6 +144,7 @@
         android:id="@+id/list_repositories"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_marginBottom="@dimen/margin_botton_recycler_view"
```

Comment:
Please set just `margin` so the 4 sides (top, left, right and bottom) will have a margin.